applications/poisson/sine/application.h

```diff
@@ -158,8 +150,9 @@ class Application : public ApplicationBase<dim, n_components, Number>
     this->param.right_hand_side = true;
 
     // SPATIAL DISCRETIZATION
-    this->param.grid.triangulation_type = TriangulationType::Distributed;
-    this->param.grid.mapping_degree     = 3;
+    this->param.grid.triangulation_type = TriangulationType::Serial;
+    this->param.grid.mapping_degree     = 2;
+    this->param.grid.element_type       = ElementType::Simplex;
```

Comment:
The default setup, which ExaDG users can see, should be Hypercube (with Multigrid) for now since the use of simplex is currently restricted.

Reply:
done!